validation/src/test/scala/hmda/validation/rules/lar/macro/Q007Spec.scala

```diff
@@ -0,0 +1,33 @@
+package hmda.validation.rules.lar.`macro`
+
+import akka.stream.scaladsl.Source
+import com.typesafe.config.ConfigFactory
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.rules.AggregateEditCheck
+import hmda.validation.rules.lar.`macro`.MacroEditTypes.LoanApplicationRegisterSource
+import org.scalacheck.Gen
+
+class Q007Spec extends MacroSpec {
+
+  val config = ConfigFactory.load()
+  val multiplier = config.getDouble("hmda.validation.macro.Q007.numOfLarsMultiplier") * 100
+
+  val multiplierGen: Gen[Int] = Gen.choose(1, multiplier.toInt)
```

Comment:
I think property specs here are a good idea 👍 . Two quick things:

- I would change the name of this generator and the variable it generates (I think `multiplier` isn't quite accurate, since it's already been multiplied by 100). Maybe `proportion`? 
- Could we also get the inverse test? I'd like to see one that shows that when badlars is > multiplier, the edit fails